Ginger/GingerCoreCommon/UIElement/ElementInfo.cs

```diff
@@ -117,6 +125,9 @@ public eElementType ElementTypeEnum
             set { mElementTypeEnum = value; }
         }
 
+        [IsSerializedForLocalRepository]
+        public ObservableList<ComboBoxOptionalValue> ComboBoxOptionalValues = new ObservableList<ComboBoxOptionalValue>();
+
```

Comment:
Element values should be saved in generic way in list called "ObservableList<String> ElementOptionalValues" so "Value" field will be for selected Value and this list for all other options, and it will be generic so also elements like ListBox and other will be using it